libDCM/hilsim.h

```diff
@@ -0,0 +1,29 @@
+// This file is part of MatrixPilot.
+//
+//    http://code.google.com/p/gentlenav/
+//
+// Copyright 2009-2011 MatrixPilot Team
+// See the AUTHORS.TXT file for a list of authors of MatrixPilot.

```

Comment:
No doubt something that one needs to be more mindful with many more of our code modules..